index.md

```diff
@@ -15,7 +15,7 @@ import org.python.util.PythonInterpreter;
 public class JythonHelloWorld {
   public static void main(String[] args) {
     try(PythonInterpreter pyInterp = new PythonInterpreter()) {
-      pyInterp.exec("print 'Hello Python World!'");
+      pyInterp.exec("print ('Hello Python World!')");
```

Comment:
Should we remove the space between`print` and the `(`

Reply:
Done